### PR TITLE
refactor: peerdas types

### PR DESCRIPTION
**Pull request by matthewkeil**
**Motivation**

Clean up peerDAS datatypes for function refactoring.

### CODE REVIEW

File: packages/beacon-node/src/chain/blocks/types.ts [hunk 1]
```diff
@@ -21,63 +21,94 @@ export enum BlockSource {
...
+ *
+ */
 /** Enum to represent where blobs come from */
 export enum BlobsSource {
   gossip = "gossip",
   api = "api",
   byRange = "req_resp_by_range",
   byRoot = "req_resp_by_root",
 }
+interface ForkBlobsInfo {
```
Comment (twoeths) [inline packages/beacon-node/src/chain/blocks/types.ts]: prefer "type" over "interface" to model types of objects

File: packages/beacon-node/src/chain/blocks/types.ts [hunk 2]
```diff
@@ -21,63 +21,94 @@ export enum BlockSource {
   byRoot = "req_resp_by_root",
 }
 
+export enum GossipedInputType {
+  block = "block",
+  blob = "blob",
+  dataColumn = "dataColumn",
```
Comment (twoeths) [inline packages/beacon-node/src/chain/blocks/types.ts]:
```suggestion
  dataColumn = "data_column",
```

I think it looks nice that way if we render on grafana

**Review by g11tech — APPROVED** [review]

File: packages/beacon-node/src/chain/blocks/types.ts (hunk 2 above)
Comment (matthewkeil) [reply packages/beacon-node/src/chain/blocks/types.ts:27]: updated

Comment (matthewkeil) [reply packages/beacon-node/src/chain/blocks/types.ts]: updated

File: packages/beacon-node/src/chain/blocks/types.ts (hunk 1 above)
Comment (matthewkeil) [reply packages/beacon-node/src/chain/blocks/types.ts]: updated

**Review by wemeetagain — APPROVED** [review]
LGTM, this appears to just add / rename types, no code change

**Pull request merged**